src/client/components/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export default class Footer extends React.Component{
```

Comment:
can we please refrain from using classes? Can we please use functional components instead?

`const Footer = () => {
    return (
     )
 }
`